ash-examples/src/lib.rs

```diff
@@ -11,11 +11,14 @@ use std::{
     borrow::Cow, cell::RefCell, default::Default, error::Error, ffi, ops::Drop, os::raw::c_char,
 };
 
-use ash::extensions::{
-    ext::debug_utils,
-    khr::{surface, swapchain},
+use ash::{
+    vk,
+    vk::{
```

Comment:
There's also (not sure if better):

```suggestion
    vk::{self,
```

Reply:
Eh, I kinda like that the syntax of the import matches the syntax of the use site.

Reply:
That's also true. In our application this pattern is a common sign for inconsistencies with imports and qualifications:



```rust
use some_crate::{self}; // Does nothing, because some_crate is already in scope? Maybe to disambiguate?

use std::sync::{self, Arc}; // Probably inconsistent use of `Arc` in one place and `sync::Mutex` in another
```